whelk-core/src/main/groovy/whelk/converter/JsonLD2RdfXml.groovy

```diff
@@ -1,45 +1,45 @@
 package whelk.converter
 
+import groovy.transform.CompileStatic
+
 import org.apache.commons.io.IOUtils
-import org.apache.jena.rdf.model.Model
 import org.apache.jena.rdf.model.ModelFactory
-import org.apache.jena.rdf.model.RDFWriter
 import whelk.Document
 import whelk.JsonLd
 import whelk.Whelk
-import whelk.component.PostgreSQLComponent
-import whelk.util.PropertyLoader
 
 import static whelk.util.Jackson.mapper
 
+@CompileStatic
 class JsonLD2RdfXml implements FormatConverter {
 
-    Map m_context = null
+    private Map context = null
 
     JsonLD2RdfXml(Whelk whelk = null) {
        if (whelk) {
-           m_context = [:]
-           m_context[JsonLd.CONTEXT_KEY] = whelk.jsonld.context
+           context = whelk.jsonld.context
        }
     }
 
     Map convert(Map originaldata, String id) {
-        readContextFromDb()
-        Map framed = JsonLd.frame(Document.BASE_URI.resolve(id).toString(), originaldata)
-
-        framed.putAll(m_context)
-        String framedString = mapper.writeValueAsString(framed)
+        var srcData = [:]
+        srcData.putAll(originaldata)
+        if (context) {
+            srcData[JsonLd.CONTEXT_KEY] = context
+        }
 
-        InputStream input = IOUtils.toInputStream(framedString)
-        Model model = ModelFactory.createDefaultModel()
-        ByteArrayOutputStream baos = new ByteArrayOutputStream()
+        var jsonldStr = mapper.writeValueAsString(srcData)
+        var input = IOUtils.toInputStream(jsonldStr)
```

Comment:
```suggestion
        var input = IOUtils.toInputStream(jsonldStr, StandardCharsets.UTF_8)
```

This seems to be the code that's used on the Jena side:
https://github.com/jsonld-java/jsonld-java/blob/da156eebe0fdd2cee34a8bb603d64979a281d389/core/src/main/java/com/github/jsonldjava/utils/JsonUtils.java#L68

Newer version of same code:
https://github.com/jsonld-java/jsonld-java/blob/f2f1cf22dc8a9c30c47ab570c7aaa8243101ad02/core/src/main/java/com/github/jsonldjava/utils/JsonUtils.java#L102